.github/workflows/run-profiling.yaml

```diff
@@ -164,6 +164,7 @@ jobs:
       ## Token can be (re)generated by a member of the UCL organisation, 
       ## the current member is the rc-softdev-admin.
       ## [10-07-2023] The current token will expire 10-07-2024
+      ## [17-07-2024] New token generated, will expire 10-07-2025
```

Comment:
I'd personally say better to just replace the current comment than add a new line, as only the latest date is relevant (and the previous comment will still be in commit history if we wanted to check previous expiration dates for some reason).